volume_zarr.py

```diff
@@ -360,6 +380,15 @@ def __getitem__(self, key):
             # first try to obtain the value from the cache
             with self._mutex:
                 value = self._values_cache[key]
+
+                # Check that the cached value is not missing data.
+                if self.__value_is_incomplete(key):
+                    expected_size_bytes = self.__expected_value_size_bytes(key)
+                    message = "%s value is incomplete, expected %i bytes but got %i bytes" % (key, expected_size_bytes, len(value))
+                    print("ERROR! " + message)
+                    del self[key] # Purge corrupt data. # Note(kbostelmann): This doesn't work ???
```

Comment:
Could remove this bit, the caveat is that the viewer will forever display a black area when painting the chunk.